docs/source/llms/transformers/index.rst

```diff
@@ -0,0 +1,279 @@
+
+.. raw:: html
+
+    <section>
+        <article class="simple-grid">
+            <div class="simple-card">
+                <a href="tutorials/text-generation/text-generation.html">
+                    <div class="header">
+                        Text Generation with Transformers
+                    </div>
+                    <p>
+                        Learn how to leverage the transformers integration with MLflow in this introductory tutorial.
+                    </p>
+                </a>
+            </div>
+            <div class="simple-card">
+                <a href="tutorials/audio-transcription/whisper.html">
+                    <div class="header">
+                        Audio Transcription with Transformers
+                    </div>
+                    <p>
+                        Learn how to leverage the Whisper Model with MLflow to generate accurate audio transcriptions.
+                    </p>
+                </a>
+            </div>
+            <div class="simple-card">
+                <a href="tutorials/translation/component-translation.html">
+                    <div class="header">
+                        Translation with Transformers
+                    </div>
+                    <p>
+                        Learn about the options for saving and loading transformers models in MLflow for customization of your workflows with a fun translation example!
+                    </p>
+                </a>
+            </div>
+            <div class="simple-card">
+                <a href="tutorials/conversational/conversational-model.html">
+                    <div class="header">
+                        Chat with Transformers
+                    </div>
+                    <p>
+                        Learn the basics of stateful chat Conversational Pipelines with Transformers and MLflow.
+                    </p>
+                </a>
+            </div>
+            <div class="simple-card">
+                <a href="tutorials/fine-tuning/transformers-fine-tuning.html">
+                    <div class="header">
+                        Fine tuning a transformers Foundation Model
+                    </div>
+                    <p>
+                        Learn how to fine-tune a transformers model using MLflow to keep track of the training process and to log a use-case-specific tuned pipeline.
+                    </p>
+                </a>
+            </div>
+            <div class="simple-card">
+                <a href="../custom-pyfunc-for-llms/index.html">
+                    <div class="header">
+                        Custom PyFunc for Transformers
+                    </div>
+                    <p>
+                        Learn how to define a custom PyFunc using transformers for advanced, state-of-the-art new models.
+                    </p>
+                </a>
+            </div>
+        </article>
+    </section>
+
+Download the Tutorial Notebooks to try them locally
```

Comment:
Unfortunately, we can't render these or properly trigger a local notebook download from within the Jupyter notebook render. nbsphinx (the plugin for sphinx that renders the .ipynb as .html) does not allow for custom JS or CSS to propagate to the rendered html (I tried). In order for us to actually have a notebook download to local on all browsers, we have to specify some custom JS to override the settings that the server (s3 website hosting) specifies for how to handle specific file types (otherwise, it will, by browser default, render the raw text of the notebook content directly in the browser). 
We might be able to come up with something different with a change to a different platform for html generation, but for now, we have to have some external to the notebook rendering mechanism that generates a link / button to download the files.